patches/@helsenorge+skjemautfyller+7.11.0-prerelease2.patch

```diff
@@ -0,0 +1,22 @@
+diff --git a/node_modules/@helsenorge/skjemautfyller/reducers/form.js b/node_modules/@helsenorge/skjemautfyller/reducers/form.js
```

Comment:
could these changes be proposed to the repo or are they too ona specific?

Reply:
I agree that would be more sensible, however, we do not have the source code  for this yet.